main.js

```diff
@@ -4,35 +4,66 @@
+            var nombreElementVal = nombreElement.getElementValue();
+            var showNameElementSelector = '#showName';
+            var $showName = $(showNameElementSelector);
+		if (nombreElementVal === '') {
 			alert("El campo nombre esta vacio");
-			e.preventDefault();
-		} 
-	};
+		} else {
+                  $showName.html('Name: ' + nombreElementVal);
+                }
+	}
+
+	var validarSexo = function (domElementToValidate) {
+            var sexoElement1 = new domElementToValidate(formulario.Radio1);
+            var sexoElement2 = new domElementToValidate(formulario.Radio2);
+            var showSexElementSelector = '#showSex';
+            var $showSex = $(showSexElementSelector);
 
-	var validarSexo = function (e) {
-		if (formulario.Radio1.checked == true || formulario.Radio2.checked == true ) {
+		if (sexoElement1.getElementCheckedProperty() === true || sexoElement2.getElementCheckedProperty() === true ) {
+                  $showSex.html('Sex: ' + sexoElement1.getElementCheckedProperty() ? sexoElement1.getElementValue() : sexoElement2.getElementValue());
 		} else {
 			alert("Debe seleccionar el sexo");
-			e.preventDefault();
 		}
 	};
 
-	var validarTerminos = function (e) {
-		if (formulario.Terminoscondiciones.checked == false) {
+	var validarTerminos = function (domElementToValidate) {
+          var terminosYCondicionesElement = new domElementToValidate(formulario.Terminoscondiciones);
+          var showTermsAndConditionsElementSelector = '#showTermsAndConditions';
+          var $showTermsAndConditions = $(showTermsAndConditionsElementSelector);
+		if (terminosYCondicionesElement.getElementCheckedProperty() === false) {
 			alert("Debe aceptar los terminos y condiciones");
-			e.preventDefault();
-		} 
+		} else {
+                  $showTermsAndConditions.html('Terms and conditions accepted');
+                }
 	};
 
 	var validar = function (e) {
-		validarNombre(e);
-		validarSexo(e);
-		validarTerminos(e);
-};
+          var domElementToValidate = function (domElement) {
+            this.elementToValidate = domElement;
+            this.$elementToValidate = $(this.elementToValidate);
+            this.elementToValidateValue = this.$elementToValidate.val();
+          };
+
+          domElementToValidate.prototype.getElementValue = function() {
+            return this.elementToValidateValue;
+          };
+
+          domElementToValidate.prototype.getElementCheckedProperty = function() {
+            return this.elementToValidate.checked;
+          };
+
+          validarNombre(domElementToValidate);
+          validarSexo(domElementToValidate);
+          validarTerminos(domElementToValidate);
+          // This event.preventDefault() prevents the browser from doing submit default action
+          // which we have fired to start the validation
+          e.preventDefault();
+        };
 
 // Eventoss
 	formulario.addEventListener("submit", validar);
```

Comment:
We request the browser to execute a submit action here and the default behavior would be that.